app-files/app.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const { getTopics } = require('../controllers/topic-controller')
+const { invalidPaths, serverErrors } = require('./errors')
+
+const app = express()
+module.exports = app
+
+app.get('/api/topics', getTopics)
+app.get('*', invalidPaths)
```

Comment:
This will only catch `get` requests to bad paths, when really we want this to catch bad paths to any method, (like `post`,`delete`,`patch` etc.). `app.all` is more applicable here